addons/a11y/src/components/A11YPanel.tsx

```diff
@@ -144,9 +152,18 @@ export class A11YPanel extends Component<A11YPanelProps, A11YPanelState> {
     });
   };
 
+  onManual = (manual: boolean) => {
+    if (manual) {
+      this.setState({
+        status: 'manual',
+      });
+    } else {
+      this.request();
+    }
+  };
+
   request = () => {
```

Comment:
@donaldpipowitch I succeeded to fix locally the issue we talk about a few days ago (https://github.com/storybookjs/storybook/pull/8883/files?file-filters%5B%5D=.snap&file-filters%5B%5D=.ts&file-filters%5B%5D=.tsx#r364712456) by removing the if-check in the request function. Can you try on your side? and add a commit in your branch if it works?

```ts
request = () => {
  const { api } = this.props;
  this.setState(
    {
      status: 'running',
    },
    () => {
      api.emit(EVENTS.REQUEST);
      // removes all elements from the redux map in store from the previous panel
      store.dispatch(clearElements());
    }
  );
};
```

I don't well understand why this `active` check is done, I played with SB examples locally and didn't find any new issue 🤷‍♂

Reply:
@gaetanmaisse thank you for having a look. This seems to fix the issue - it now works for me locally.

I'm unsure about `active` as well. I mostly left this untouched. But there is an old comment which says that it might can be removed completely?
https://github.com/storybookjs/storybook/blob/2da3a4e1dd5a5f489624ebb0949e0b941bfd31e9/addons/a11y/src/components/A11YPanel.tsx#L100

For now I just added your fix and kept other usages of `active`.

Reply:
I tried to simply remove all `active` usages and it breaks everything so I think we need to keep it for now.